src/sentry/api/endpoints/organization_member_invite/index.py

```diff
@@ -0,0 +1,212 @@
+
+        is_member = not request.access.has_scope("member:admin") and (
+            request.access.has_scope("member:invite")
+        )
+
+        validator = OrganizationMemberInviteRequestValidator(
+            data=request.data,
+            context={
+                "organization": organization,
+                "allowed_roles": allowed_roles,
+                "allow_retired_roles": not features.has("organizations:team-roles", organization),
+                "is_integration_token": request.access.is_integration_token,
+                "is_member": is_member,
+                "actor": request.user,
+            },
+        )
+
+        if not validator.is_valid():
+            return Response(validator.errors, status=400)
+
+        result = validator.validated_data
+
+        if ratelimits.for_organization_member_invite(
+            organization=organization,
+            email=result["email"],
+            user=request.user,
+            auth=request.auth,
+        ):
+            metrics.incr(
+                "member-invite.attempt",
+                instance="rate_limited",
+                skip_internal=True,
+                sample_rate=1.0,
+            )
+            return Response({"detail": ERR_RATE_LIMITED}, status=429)
+
+        omi = _create_invite_object(request, organization, result, is_request=False)
+
+        referrer = request.query_params.get("referrer")
+        omi.send_invite_email(referrer)
+        member_invited.send_robust(
+            member=omi, user=request.user, sender=self, referrer=request.data.get("referrer")
+        )
+
+        return Response(serialize(omi), status=201)
+
+    def _request_to_invite_member(self, request: Request, organization) -> Response:
+        validator = OrganizationMemberInviteRequestValidator(
+            data=request.data,
+            context={
+                "organization": organization,
+                "allow_retired_roles": not features.has("organizations:team-roles", organization),
+                "allowed_roles": roles.get_all(),
+                "actor": request.user,
+            },
+        )
+        if not validator.is_valid():
+            return Response(validator.errors, status=400)
+        result = validator.validated_data
+
+        omi = _create_invite_object(request, organization, result, is_request=True)
+
+        async_send_notification(InviteRequestNotification, omi, request.user)
+        return Response(serialize(omi), status=201)
+
+    def get(self, request: Request, organization) -> Response:
+        """
+        List all organization member invites.
+        """
+        queryset = OrganizationMemberInvite.objects.filter(organization=organization).order_by(
+            "invite_status", "email"
+        )
+        if not request.access.has_scope("member:write"):
+            queryset = queryset.filter(invite_status=InviteStatus.APPROVED.value)
```

Comment:
forgot to comment
it seems like everyone can see the requests
https://github.com/getsentry/sentry/blob/751c930ad352341d8f976a7ac5582b46fd1770a3/static/app/views/settings/organizationMembers/organizationMembersList.tsx#L332-L356